tests/bsim/bluetooth/audio/src/main.c

```diff
@@ -56,6 +58,8 @@ bst_test_install_t test_installers[] = {
 	test_has_client_install,
 	test_ias_install,
 	test_ias_client_install,
+	test_tmap_install,
+	test_tmap_client_install,
```

Comment:
Suggest to rename these to `tmap_central` and `tmap_peripheral` or `tmap_client` and `tmap_server`